source/docs/software/odometry.rst

```diff
@@ -0,0 +1,31 @@
+=============
+Odometry
+=============
+Odometry is a form of localization that uses data from sensors
+like encoders to derive an estimated position relative to a
+starting point. Localization is a means for being bale to locate
+the position of the bot at some point in time. Odometry is especially
+useful in autonomous programs because it allows for easier implementation
+of different tasks on the field due to understanding one's position
+
+Pose
+=============
+We refer to pose, which is the position of some body (like a bot),
+normally in the context two-dimensional space,
+as the movement of the robot is generally constrained to a
+single plane. We notate the robot's pose as :math:`\vec{x}`.
+A pose contains three entries: the x-coordinate, the y-coordinate,
```

Comment:
I'd change this to say something like "A pose contains two entries, the robot's position and heading; position is typically measured with Cartesian coordinates, and so a pose can also be represeneted with :math:`x`, :math:`y`, and :math:`\theta`".

Reply:
I think I can remove the redundant double conjunction "and so":
A pose contains two entries: the robot's position and heading; position is generally in Cartesian coordinates, so the pose can be represented with :math:`x`, :math:`y`, and :math:`theta`.

Reply:
looks good